addons/libs/packets/fields.lua

```diff
@@ -2098,6 +2098,8 @@ types.shop_item = L{
     {ctype='unsigned int',      label='Price',              fn=gil},            -- 00
     {ctype='unsigned short',    label='Item',               fn=item},           -- 04
     {ctype='unsigned short',    label='Shop Slot'},                             -- 08
+	{ctype='unsigned short',    label='Craft Skill'},                           -- Zero on normal shops, has values that correlate to res\skills.
```

Comment:
You're using tabs. Please switch to 4 spaces (in this and all of your pull requests).